### PR TITLE
Uploaded Successfully In Assignment 03

### PR DESCRIPTION
**Pull request by Ashwin2812**

### CODE REVIEW

**Review by pritikekan — COMMENTED** [review]
you did not send  complete assignment

**Review by pooja-nagare — COMMENTED** [review]
you have not done MI-MEAN.html and Contact-Us.html

File: Ashwin/HTML/Assignment_03_12042021/AshwinA3_2.html [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+        color: blue;
+    }
+    a:visited{
+        color: darkred;
+    }
+</style>
+<body>
+    <div style="text-align: center;">
+        <h1>AmberSoft</h1><br>
+        <a href="#" >Home </a> |
```
Comment (pooja-nagare) [inline Ashwin/HTML/Assignment_03_12042021/AshwinA3_2.html:24]: make changes <a href="AshwinA3_2.html">Home</a>

**Review by Ashwin2812 — COMMENTED** [review]
Completed

**Pull request closed without merging**